app/src/main/res/values-en/strings.xml

```diff
@@ -564,7 +564,7 @@
     <string name="start">GOT IT</string>
     <string name="contact">Contact</string>
     <string name="contact_summary">Contact the team</string>
-    <string name="off_mail" translatable="false">mailto:contact@openfoodfacts.org</string>
+    <string name="off_mail" translatable="false">mailto:<u>contact@openfoodfacts.org</u></string>
```

Comment:
Why? This needs to be a valid URL.

Reply:
It still is. I underlined it to follow the common(universal) notation.
Should I undo that?

Reply:
If it is still clickable, nope.